### ISSUE TITLE
How to add a dropout layer to a specified functional model?

### ISSUE THREAD

**Issue by TillLindemann** [keras-team/keras#9176]
### environment: 
       latest keras version,
       latest tensorflow version,
       python3.6,
       ubuntu17.10
### problems:
I want to add a dropout layer to my functional model and I didn't see any examples,so I don't know how to do it.
### below is my code:
```python
house_input = Input(shape=(None,None,33),dtype='float32')
house_middle = LSTM(units,return_sequences = True,activation=self.activation_f,kernel_initializer=self.initializer)(house_input)
house_output = LSTM(units,return_sequences = True,activation=self.activation_f,kernel_initializer=self.initializer)(house_middle)
#dropout layer should be added here.
finale_output = Dense(1)(house_output)
model =Model(inputs=house_input,outputs=finale_output)
```
### some explanations about my code:
I want to build a multi-layer RNN(LSTM Cell),the data coming in is the shape of (batch_size,timesteps,33).
### where to add:
I want to add to the second LSTM layer BELOW,How I suppose to do it?
Any suggestions will be appreciated!
Thanks

**Comment by tomastheod-ITI** [comment]
Import the Dropout layer with `from keras.layers import Dropout` and then use it like any other layer:
``` python
house_input = Input(shape=(None, None, 33), dtype='float32')
house_middle = LSTM(units, return_sequences = True, activation=self.activation_f, kernel_initializer=self.initializer)(house_input)
house_output = LSTM(units, return_sequences = True, activation=self.activation_f, kernel_initializer=self.initializer)(house_middle)
#dropout layer should be added here.
house_output_dropout = Dropout(0.25)(house_output)
finale_output = Dense(1)(house_output_dropout)
model =Model(inputs=house_input, outputs=finale_output)
```

**Comment by TillLindemann** [comment]
@tomastheod-ITI  OK，it works,Thank you for helping me
